compiler/rustc_typeck/src/check/generator_interior.rs

```diff
@@ -22,6 +22,10 @@ use tracing::debug;
 
 mod drop_ranges;
 
+// FIXME(eholk): This flag is here to give a quick way to disable drop tracking in case we find
+// unexpected breakages while it's still new. It should be removed before too long.
```

Comment:
Maybe point to the issue this works around here?

Reply:
Done